### PR TITLE
optimized response page

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Dev:

### CODE REVIEW

File: assets/scripts/admin/tokens.js [hunk 1]
```diff
@@ -137,8 +137,8 @@ function submitEditToken(){
             if (result.success) {
                 $modal.hide();
                 $('body').removeClass('modal-open');
+                $('body').removeAttr('style');
                 $('.modal-backdrop').remove();
-                window.LS.ajaxAlerts(result.success, 'success');
```
Comment (twilligls) [inline assets/scripts/admin/tokens.js:141]: Why did you remove that? I cannot see a replacement success alert somewhere.

**Review by twilligls — CHANGES_REQUESTED** [review]

**Comment by twilligls** [comment]
At least one test regarding your changes is failing now

File: application/views/responses/browseindex_view.php [hunk 2]
```diff
@@ -13,53 +13,55 @@
...
+            [
+                'num_completed_answers' => $num_completed_answers,
+                'num_total_answers' => $num_total_answers,
+                'num_total_answers' => $num_total_answers,
+            ],
+            true
+        );
+        ?>
     </div>
-    <?php if (isset($with_token)) : ?>
```
[the remark this answers is not in the document]
Comment (twilligls) [reply application/views/responses/browseindex_view.php:38]: I mean in line 9

File: application/views/responses/browseindex_view.php [hunk 3]
```diff
@@ -13,53 +13,55 @@
...
-        <h3><?php eT("Survey participant summary"); ?></h3>
-        <div class="row">
-            <div class="col-12 content-right">
-                <table class='ls-statisticssummary table'>
-                    <tbody>
-                        <tr>
-                            <th><?php eT("Total invitations sent"); ?></th>
-                            <td><?php echo $tokeninfo['sent']; ?></td>
```
[the remark this answers is not in the document]
Comment (twilligls) [reply application/views/responses/browseindex_view.php:46]: I mean in line 8

File: application/views/responses/browseindex_view.php [hunk 4]
```diff
@@ -5,61 +5,62 @@
...
-        </div>
+<div class='side-body survey-response-page <?php echo getSideBodyClass(true); ?>'>
+    <h1> <?= gT("Survey responses") ?> </h1>
+    <div class="mt-4">
+        <?php echo $this->renderPartial(
+            'partial/responseSummary',
+            [
+                'num_completed_answers' => $num_completed_answers,
+                'num_total_answers' => $num_total_answers,
+                'num_total_answers' => $num_total_answers,
```
Comment (tiborpacalat) [inline application/views/responses/browseindex_view.php:22]: Here is an error probably due to copy paste - you are missing incomplete responses.

**Review by tiborpacalat — CHANGES_REQUESTED** [review]

**Pull request merged**